src/ripple/app/tx/apply.h

```diff
@@ -56,38 +65,57 @@ checkValidity(HashRouter& router,
 
 
 /** Sets the validity of a given transaction in the cache.
-    Use with extreme care.
+
+    @warn Use with extreme care.
```

Comment:
Oops. The correct tag is `@warning`. I don't see it rendering much differently, but oh well.